File: 4107-find-missing-elements/4107-find-missing-elements.cs
public class Solution {
    public IList<int> FindMissingElements(int[] nums) {
        int n = nums.Length;
        var vis = new bool[101];

        int min = Int32.MaxValue;
        int max = Int32.MinValue;
        for (var g = 0; g < n; g++) {
            min = Math.Min(min, nums[g]);
            max = Math.Max(max, nums[g]);
            vis[nums[g]] = true;
        }

        var ans = new List<int>();
        for (int i = min; i < max; ++i) {
            if (!vis[i])
            ans.Add(i);
        }
        return ans;
    }
}

File: 4107-find-missing-elements/4107-find-missing-elements.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findMissingElements = function(nums) {
    let n = nums.length;
    let max = -1;
    let min = 1234567889;
    
    const vis = Array(101).fill(false);
    for (let w = 0; w < n; w++) {
        max = Math.max(max, nums[w]);
        min = Math.min(min, nums[w]);
        vis[nums[w]] = true;
    }
    const arr = []
    for (let i = min; i < max; ++i) {
        if (!vis[i])
            arr.push(i);
    }
    return arr;
};